src/sentry/static/sentry/app/routes.jsx

```diff
@@ -795,14 +791,38 @@ function routes() {
               component={errorHandler(LazyLoad)}
             />
           </Route>
-
+          <Route
+            path="/organizations/:orgId/issues/"
+            componentPromise={() =>
+              import(/* webpackChunkName: "OrganizationStreamContainer" */ './views/organizationStream/container')}
+            component={errorHandler(LazyLoad)}
+          >
+            <IndexRoute
+              componentPromise={() =>
+                import(/* webpackChunkName: "OrganizationStreamOverview" */ './views/organizationStream/overview')}
```

Comment:
I think the name is good - Is the stream something we'll want to keep codesplit once it's live on prod? We probably want it in the main bundle (and codesplit the older stream).

Reply:
Yeah once org issues are out they should probably join the main bundle but for now I didn't want to bog down customers with code they won't use. I added APP-992 to track the removal of the code split.